src/helpers/dataframe/sort.ts

```diff
@@ -19,7 +20,9 @@ export function sortableDataFrame<M extends Obj, C extends Obj>(
     }
     return sortable === false || sortable === undefined // If the column is not in sortableColumns, it should not be sortable
   })) {
-    // TODO(SL): we should return a clone of the data frame (and we should provide a helper function to clone a dataframe).
```

Comment:
I don't know if we should remove this comment. It depends on whether we expect this function to always return a new (deeply cloned) object or not.